tests/test_main_window.py

```diff
@@ -0,0 +1,70 @@
+import pytest
+from qtpy.QtCore import Qt
+from qtpy.QtWidgets import QDockWidget, QWidget
+
+from lucid import LucidMainWindow
+
+
+@pytest.fixture(scope='function')
+def main_window(qtbot):
+    main_window = LucidMainWindow()
+    qtbot.addWidget(main_window)
+    return main_window
+
+
+def test_add_multiple_docks(main_window):
+    first_dock = QDockWidget()
+    second_dock = QDockWidget()
```

Comment:
I think you need to add the `QDockWidget` objects to the qtbot to ensure proper cleanup.